docs/users.rst

```diff
@@ -728,28 +728,37 @@ include the fetched artifacts in addition to the installed RPMs.
 Image tags
 ----------
 
-The output from atomic-reactor includes container images tagged into a
-registry (or Pulp, if Pulp integration is enabled). In addition, when
-multi-platform builds are enabled each set of images will be grouped
-into a manifest list, which itself is tagged.
+OSBS's atomic-reactor pushes the new container image to the container
+registry (or Pulp, if Pulp integration is enabled) and updates various tag
+references in the registry. In addition, when multi-platform builds are
+enabled, atomic-reactor groups each set of images into a manifest list
+and tags that manifest list.
 
-While the repository name is specified by the ``name`` label in the
-Dockerfile, the tags used within the repository are:
+OSBS determines the name of the repository from the ``name`` label in the
+Dockerfile. There are three categories of tags that OSBS creates when
+tagging the resulting image in the registry:
 
-- a unique tag including the timestamp (this tag is the only tag
-  applied for scratch builds)
-- ``{version}-{release}`` (the ``version`` and ``release`` labels
-  together)
+* A "unique" tag: This tag includes the timestamp of when the image was
```

Comment:
Maybe we could make tag type more visible by italics+bold font?

Reply:
Good idea. I've made the tags bold and amended this pull request.